src/coreclr/jit/ssabuilder.cpp

```diff
@@ -799,8 +805,12 @@ void SsaBuilder::RenameDef(GenTreeOp* asgNode, BasicBlock* block)
 
         lclNode->SetSsaNum(SsaConfig::RESERVED_SSA_NUM);
     }
+    else if (defNode->OperIs(GT_CALL))
```

Comment:
I'm a bit puzzled why we can just return here instead of running the rest of this method.

Reply:
There was a discussion about it at https://github.com/dotnet/runtime/pull/64130#discussion_r821848726.